### PR TITLE
Add ci support for baremetal networking repos

### PR DESCRIPTION
**Pull request by cybertron**
Add necessary code in run_ci.sh to test the baremetal networking
repos.

### CODE REVIEW

File: run_ci.sh [hunk 1]
```diff
@@ -88,6 +88,9 @@ if [ -d "/home/notstack/ironic-inspector-image" ] ; then
     export IRONIC_INSPECTOR_LOCAL_IMAGE=https://github.com/metal3-io/ironic-inspector-image
     export UPSTREAM_IRONIC=true
 fi
+if [ -d "/home/notstack/baremetal-runtimecfg" ] ; then
+    export BAREMETAL_RUNTIMECFG_LOCAL_IMAGE=https://github.com/metal3-io/ironic-inspector-image
```
Comment (derekhiggins) [inline run_ci.sh]: copy/paste error here in the URL

Comment (cybertron) [reply run_ci.sh]: Oops, fixed. Thanks!

**Comment by derekhiggins** [comment]
lgtm

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1439/

**Pull request merged**